accounts/migrations/0001_initial.py

```diff
@@ -11,36 +11,123 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('auth', '0009_alter_user_last_name_max_length'),
+        ("auth", "0009_alter_user_last_name_max_length"),
```

Comment:
I would avoid format the migrations, what do you think? at the end it's an autogenerated file by django.

Reply:
I also agree width Jordi.

Reply:
@jbagot, yep, makes sense. I just reverted the code format on all migrations files.